version.go

```diff
@@ -17,14 +19,15 @@ var (
 
 // Version contains Argo version information
 type Version struct {
-	Version      string
-	BuildDate    string
-	GitCommit    string
-	GitTag       string
-	GitTreeState string
-	GoVersion    string
-	Compiler     string
-	Platform     string
+	Version        string
+	BuildDate      string
+	GitCommit      string
+	GitTag         string
+	GitTreeState   string
+	GoVersion      string
+	Compiler       string
+	Platform       string
+	KsonnetVersion string
```

Comment:
Let's not change the version.go at the root (i.e. i don't think we should be add KsonnetVersion here). This will add ksonnet version to all our CLIs, but really we only want this from the API server.

Reply:
Just to add, the reason we don't want this behavior, is because we do not want to make ksonnet a requirement when someone runs `argocd version` from the CLI. Not everyone will have ksonnet installed on their system.